authentication/utils.py

```diff
@@ -71,3 +71,22 @@ def is_user_email_blocked(email):
     """Returns the user's email blocked status"""
     hash_object = hashlib.md5(email.lower().encode("utf-8"))
     return BlockList.objects.filter(hashed_email=hash_object.hexdigest()).exists()
+
+
+def block_user_email(email):
+    """Blocked the user's email if provided"""
+    msg = None
+    if email:
+        hash_object = hashlib.md5(email.lower().encode("utf-8"))
```

Comment:
Can we create a utility function for this step, I think it's been used a lot of times now.

Reply:
I am now using in `utils.py'.